.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
We don't want the `.idea` folder committed to git.

For easier exclusion, you may want to ignore the `.idea` folder in your global `.gitignore`

Reply:
Sure I can do that, if we get the other problems solved. I just thought this was abandoned, so that's why I commited my `.idea/` folders.

Globally though this is useful to me.